DOKS-wordpress/README.md

```diff
@@ -150,6 +152,255 @@ Finally, you need to setup the trusted sources between your MySQL database and y
+  #   value can be: "kubernetes.io/storage-node,kubernetes.io/nfs-node"
+  # nfsServerNodeAffinity: "kubernetes.io/storage-node,kubernetes.io/nfs-node"
+  #
+  # nfsHookConfigMap represent the ConfigMap name to be used for hook configuration.
+  # By default, nfsHookConfigMap is set to empty.
+  # If nfsHookConfigMap is set then chart will mount the configmap using volume, named `hook-config`
+  nfsHookConfigMap: ""
+
+nfsStorageClass:
+  name: openebs-kernel-nfs
+  reclaimPolicy: Delete
+  nfsServerType: kernel
+  isDefaultClass: false
+  backendStorageClass: "do-block-storage"
+  # The customServerConfig key passes a custom /etc/exports configuration to
+  # the NFS servers created using this StorageClass.
+  # The configuration settings are not validated, and can lead to security
+  # vulnerability.
+  # USING THIS IS NOT RECOMMENDED
+  customServerConfig: ""
+  # leaseTime defines the renewal period(in seconds) for client state
+  leaseTime:
+  # graceTime defines the recovery period(in seconds) to reclaim locks
+  # setting graceTime and leaseTime lower will reduce the io pause time during nfs server restart
+  graceTime:
+  # filePermissions defines the file ownership and mode specifications
+  # for the NFS server's shared filesystem volume.
+  # File permission changes are applied recursively if the root of the
+  # volume's filesystem does not match the specified value.
+  # For more information: https://github.com/openebs/dynamic-nfs-provisioner/blob/develop/docs/tutorial/file-permissions.md
+  filePermissions: {}
+  # The UID value is used to set the user-owner of NFS shared directory. Only valid
+  # UIDs are accepted.
+  # The ownership change is carried out recursively down the directory tree.
+  #  UID: ""
+  # The GID value is used to set the group-owner of NFS shared directory. Only valid
+  # GIDs are accepted.
+  # The ownership change is carried out recursively down the directory tree.
+  #  GID: ""
+  # The mode value is used to set the file mode of NFS shared directory. Both octals (e.g. 0744)
+  # and incremental/decremental (e.g. "u+r", "o+rw") values are accepted.
+  # The file mode change is carried out recursively down the directory tree.
+  #  mode: ""
+
+  # nfsServerResources defines the NFS server resource requests and limits
+  # Usually, below request and limits are good enough for NFS Server to work
+  # seamlessly(IOs will be taken care by kerner space process i.e nfsd).
+  nfsServerResources: {}
+  #  requests:
+  #    memory: 50Mi
+  #    cpu: 50m
+  #  limits:
+  #    memory: 100Mi
+  #    cpu: 100m
+
+nfsServer:
+  useClusterIP: "true"
+  imagePullSecret: ""
+
+analytics:
+  enabled: "true"
+```
+
+**Note:**
+
+Most of the overrides are self-explanatory and can be customized. Please visit [openebs nfs provisioner helm values](https://github.com/openebs/dynamic-nfs-provisioner/blob/develop/deploy/helm/charts/values.yaml) for more details.
+
+Finally, install the chart using Helm:
```

Comment:
Before installing the chart, you need to tell the user to clone the repo (if not already), and change directory context (or provide the absolute path to the Helm values file).